hack/jenkins/linux_integration_tests_container.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# This script runs the integration tests on a Linux machine for the Virtualbox Driver
+
+# The script expects the following env variables:
+# MINIKUBE_LOCATION: GIT_COMMIT from upstream build.
+# COMMIT: Actual commit ID from upstream build
+# EXTRA_BUILD_ARGS (optional): Extra args to be passed into the minikube integrations tests
+# access_token: The Github API access token. Injected by the Jenkins credential provider.
+
+
+
+set -e
+result=0
+JOB_NAME="Linux-Container"
+
+NAME=devshell-"$(cat /dev/urandom | tr -cd 'a-f0-9' | head -c 32)"
```

Comment:
You don't need to use the --name parameter, it might be a bit more robust to grab the generated container id like:

cid=$(docker run -d ...) and avoid passing in --name everywhere.